.github/workflows/create_docker_image.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Dockerfile on DockerHub
+on: 
+  push: 
+    branches:
+      - main
+    paths: 
+      - 'Dockerfile'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: '0'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.33.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        WITH_V: true
+        RELEASE_BRANCHES: main
+      id: bump
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+      with:
+        tag_name: ${{ steps.bump.outputs.new_tag }}
+        release_name: ${{ steps.bump.outputs.new_tag }}
+        body: |
+          Changes in this Release
+          - Rebuilt Docker image and published to DockerHub with new tag
+        draft: false
+        prerelease: false
+    - name: Publish to Registry
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: doristaosiqi/dockerfile # change this to your DockerHub username and repository
```

Comment:
Can you change the repo name to match our project repo name?

Reply:
lemme try

Reply:
so it seems that i cannot change the repo names on docker hub so im creating a new repo, im naming it collision_prediction